### PR TITLE
feat: device power toggle in top navigation

### LINKED ISSUE #729: Power Button on Top Bar
### Is your feature request related to a problem? Please describe

May sound stupid, but clicking the sidebar menu, then power, then printer is an unnecessary few extra steps every time you want to power on/off the machine. 

### Describe the solution you'd like

Instead, would it be possible to have the Moonraker printer power feature/button moved to the Top Bar for easier access? Just a simple one click on/off icon right next to the user icon would be perfect.

For other power devices they could reside in the current menu since they (_probably_) aren't used as often.

### Describe alternatives you've considered

_N/A_

### Additional information

Definitely a very minor complaint, but I figured I'd put it out there. Thank you for the amazing interface, I love Fluidd very much!

### PR DESCRIPTION
**Pull request by matmen**
Closes #729 - Supersedes #736
image
image

Also swaps the power switch icon with toggle switch icons for more intuitive UX:
image image
image image

Signed-off-by: Mathis Mensing <matmen@dreadful.tech>

### CODE REVIEW

File: src/locales/en.yaml [hunk 1]
```diff
@@ -240,6 +240,7 @@ app:
       'off': 'Off'
       password: Password
       power: Power
+      power_toggle_caption: Toggle power for device %{device}
```
Comment (pedrolamas) [inline src/locales/en.yaml:243]:
```suggestion
      power_toggle_caption: Toggle %{device} power
```

**Review by pedrolamas — APPROVED** [review]
This is great!! 🙂

I've suggested an alternative translation for the tooltip, seems to me more to the point.

Now that I've had the chance to play with this a bit more, I do wonder if the button should have the same style as the others (white and round) instead of using the same layout as the Emergency Stop...

**Comment by Martyfree123** [comment]
You guys are so fast, thanks so much!

**Comment by Martyfree123** [comment]
Personally I think it would be nice if it retains the colour matching but maybe uses the standard power icon?

**Comment by matmen** [comment]
> Now that I've had the chance to play with this a bit more, I do wonder if the button should have the same style as the others (white and round) instead of using the same layout as the Emergency Stop...

> Personally I think it would be nice if it retains the colour matching but maybe uses the standard power icon?

Both suggestions sound good to me, I'll implement those later 😉

**Comment by matmen** [comment]
> Personally I think it would be nice if it retains the colour matching but maybe uses the standard power icon?

@Martyfree123 Just to clarify - the green in the icon is only because of my fluidd theme. Would you like the icon to be colored (green and red?) accordingly or for it to just be white (or theme colored)?

**Comment by Martyfree123** [comment]
I would prefer it was theme colourd. :)

**Review by pedrolamas — APPROVED** [review]

**Pull request merged**